image/spec/README.md

```diff
@@ -23,6 +23,15 @@ and in [OCI Image Format Specification](https://github.com/opencontainers/image-
 
 All 1.X versions are compatible with older ones.
 
+### [v1.3](https://github.com/moby/moby/blob/ff46faa238f7ea7e8deddfde6c46cdde07fd680c/image/spec/v1.3.md)
+
+* Implemented in Docker v25.0 (to be released)
+* The official spec document was written in August 2023 ([#46381](https://github.com/moby/moby/pull/46381))
+
+Changes:
+
+* `StartInterval` was added to the `Healthcheck` struct in the Image JSON
```

Comment:
Thinking of that; perhaps we should drop the link to the "official spec document was written", and just limit to "chances"

This line could use a link to the PR that implemented it (but not even sure if we need to)